src/anal_ghidra.cpp

```diff
@@ -823,6 +823,23 @@ static void sleigh_esil(RAnal *a, RAnalOp *anal_op, ut64 addr, const ut8 *data,
 			return false;
 	};
 
+	auto print_operand = [&esil_stack, &ss](const PcodeOperand *arg, int offset = 0) -> bool {
+		if(arg->is_unique()) {
+			int index = index_of_unique(esil_stack, arg);
+			if(-1 == index)
+				throw LowlevelError(
+				    "print_if_unique: Can't find required unique varnodes in stack.");
+
+			ss << index + offset << ",PICK";
```

Comment:
i think @condret will complain about this PICK :D

Reply:
Yea I am going to try to remove the PICKs. Also there is something still off with the FP ops. Well not the operations but assigning to registers doesn't work properly. gonna convert to draft

Reply:
Actually there's no PICK in the esil vm, but it doesn't seems to complain when evaluating expressions with unknown operations
![Screenshot 2020-12-22 at 01 57 06](https://user-images.githubusercontent.com/6431515/102836181-01243580-43f9-11eb-9d81-ad736257110a.png)